ovn/ovn-nb.ovsschema

```diff
@@ -269,6 +274,16 @@
                     "type": {"key": "string", "value": "string",
                              "min": 0, "max": "unlimited"}}},
             "isRoot": false},
+        "Logical_Router_Policy": {
+            "columns": {
+                "priority": {"type": {"key": {"type": "integer",
+                                              "minInteger": 0,
+                                              "maxInteger": 32767}}},
+                "match": {"type": "string"},
+                "action": {"type": {"key": {"type": "string",
+                                            "enum": ["set", ["allow", "drop", "reroute"]]}}},
+                "nexthop": {"type": {"key": "string", "min": 0, "max": 1}}},
```

Comment:
Sorry, just for my understanding.
Next_hop here is the IP for reroute right?

Reply:
Correct. Do you think we should rename this to reroute_ip?

Reply:
Sorry for late reply. Lets see what upstream folks say.
I am fine with next_hop as well.